web/src/pages/about.mdx

```diff
@@ -0,0 +1,59 @@
+
+import { Box, Button, Grid, Heading, Label } from '@theme-ui/components';
+import { navigate } from 'gatsby';
+import Img from 'gatsby-image';
+import { AboutList } from '../components/about-page/about-list';
+import { Container } from '../components/container';
+import { Layout } from '../components/layouts';
+
+# TITLE
+
+<Heading
+    as="h1"
+    sx={{
+        fontWeight: 'normal',
+        width: ['100%', '60%'],
+        my: 5,
+    }}
+>
```

Comment:
@cobraz Do you want smth like that?
```
---
title: Slik forvandler vi i Bjerk dine gode ideer om til unike opplevelser.
---

<Heading>{props.pageContext.frontmatter.title}</h1>
...
```

Reply:
Let's review. For blog articles, I would imagine `<Heading />` isn't needed, as it would be a part of an «article component»?

I don't see a reason to use frontmatter for pages, given that we have to reference it like you've shown there.